gobreaker.go

```diff
@@ -98,13 +98,19 @@ func (c *Counts) clear() {
 // Default ReadyToTrip returns true when the number of consecutive failures is more than 5.
 //
 // OnStateChange is called whenever the state of the CircuitBreaker changes.
+//
+// IsSuccessful is called with the error returned from the request, if not nil.
```

Comment:
Please modify the comments because IsSuccessful returns the opposite value of ShouldTrip.